lti_consumer/lti_consumer.py

```diff
@@ -645,6 +689,25 @@ def lis_result_sourcedid(self):
             user_id=self.user_id
         )
 
+    @property
```

Comment:
Issue here: these handler urls return different URLs from LMS and Studio.
We need to use the course blocks API to retrieve the LMS urls of each handler.

The downside is that the LTI integration will only work from the LMS.